server/src/main/java/eu/fasten/server/plugins/kafka/FastenKafkaPlugin.java

```diff
@@ -432,22 +469,22 @@ private void skipPartitionOffsets() {
             return;
         }
         // Note that this assumes that the consumer is subscribed to one topic only
-        for (PartitionInfo p : this.connection.partitionsFor(topics.get(0))) {
+        for (PartitionInfo p : this.connNorm.partitionsFor(topics.get(0))) {
```

Comment:
Either this method is incomplete (i.e., also add handling for connPrio), or it is not used at all, in which case, we should remove it altogether.